channeldb/graph.go

```diff
@@ -2092,13 +2095,19 @@ func (c *ChannelGraph) FilterKnownChanIDs(chanIDs []uint64) ([]uint64, error) {
 
 			// If the edge is a known zombie, skip it.
 			if zombieIndex != nil {
-				isZombie, _, _ := isZombieEdge(zombieIndex, cid)
-				if isZombie {
+				isZombie, _, _ := isZombieEdge(
+					zombieIndex, scid,
+				)
+				if isZombie && isZombieChan(
```

Comment:
Ah yeah, this makes sense, so scratch the idea about the cache I had. I had envisioned a look up here each time to double check our timestamp with theirs. 

Thinking about it a bit more, perhaps we should do some sort of sanity check? Otherwise, they can send bogus timestamps and make us request everything all over again.

Reply:
How would such a sanity check work?  I imagine a malicious peer could trivially defeat any check by setting timestamps to a recent time.